x11-terms/gnome-terminal/files/gnome-terminal-3.44.1-fix-missing-wexitcode.patch

```diff
@@ -0,0 +1,14 @@
+--- a/src/terminal.cc
```

Comment:
Plesse put the commentary from the commit message at the top of the patch too, so that it's there without having to dive in commit logs.